src/controllers/todo.controller.ts

```diff
@@ -1,26 +1,16 @@
 import { post, param, get, put, patch, del, HttpErrors } from '@loopback/rest';
-import { TodoSchema, Todo, TodoItem } from '../models';
-import { repository, EntityCrudRepository } from '@loopback/repository';
+import { TodoSchema, Todo } from '../models';
```

Comment:
nitpick: our coding style has no spaces inside/around curly braces

Reply:
I'm not sure why Prettier isn't picking up our configuration and formatting it accordingly...